src/components/HomeViewUsers/index.css

```diff
@@ -0,0 +1,38 @@
+.img__user {
```

Comment:
this is not correct naming according to BEM.
We should start wit the component name.

In this case a name like `user__profile_pic` may be a good option